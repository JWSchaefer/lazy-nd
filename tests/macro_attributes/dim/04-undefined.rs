use lazy_nd::lazy_nd;

#[lazy_nd(dim = D)]
struct Test {}

fn main() {}
