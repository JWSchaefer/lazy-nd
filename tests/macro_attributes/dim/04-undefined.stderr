error: Failed to match the provided generic `D`. Expected `< .. , const D : usize, .. >`.
 --> tests/struct_attributes/dim/04-undefined.rs:3:1
  |
3 | #[lazy_nd(dim = D)]
  | ^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `lazy_nd` (in Nightly builds, run with -Z macro-backtrace for more info)
